Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -105,7 +105,17 @@ public extension String {
 		return String(first)
 	}
 
-	#if canImport(Foundation)
+    /// SwifterSwift: First character of string uppercased(if applicable) while keeping the original string.
+    ///
+    ///        "hello world".firstCharacterUppercased -> "Hello world"
+    ///        "".firstCharacterUppercased -> ""
+    ///
+    public var firstCharacterUppercased: String {
+        guard let first = first else { return "" }
```

Comment:
Use `return self`, not `return ""`. I know that it's most likely to be the same, but you may catch some edge-cases where there's an unfinished unicode character or something.